app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db")
+        self.table_name = "actors"
+
+    def all(self) -> None:
+        actors_cursor = self._connection.execute(
+            f"select id, first_name, last_name from {self.table_name}"
```

Comment:
SQL syntax is better written in capital letters
please fix it here and it other places